spec/requests/movies_spec.rb

```diff
@@ -215,7 +215,7 @@
     end
 
     let(:name) { Faker::Movie.title }
-    let(:duration) { 120 }
+    let(:duration) { Faker::Number.number(digits: 2) }
```

Comment:
You do not need to redefine the duration and name in every context
If you do not need a variable to be anything different than what was defined in the top describe/context block than you can just reuse it, you have access to it in every nested describe/context block

Reply:
I did a lot of changes so i'd really appreciate if you could look at it again and if everything's okey i'd do the same to movie and seance specs from the other PR

Reply:
Thanks for all the comments!